statsmodels/stats/_lilliefors.py

```diff
@@ -166,6 +166,7 @@ def get_lilliefors_table(dist='norm'):
 
 
         # also build a table for larger sample sizes
+
```

Comment:
@jbrockmendel I think we actually should've removed lines here. flake8 can be confusing/finicky

Reply:
I made a quick fix is #5335 ... let's see if I know what I'm doing.